File: src/app/entities/notifications/notification.ts
export class Notification {
    id: number;
    status: string;
    deleted: boolean;
    type: string;
    text: string;
    timeStamp: string;
    user: string;

    

    constructor(
        id: number,
        status: string,
        deleted: boolean,
        type: string, 
        text: string,
        timeStamp: string,
        user: string
        )       
    {
        this.id = id;
        this.status = status;
        this.deleted = deleted;
        this.type = type;
        this.text = text;
        this.timeStamp = timeStamp;
        this.user = user;
        
    }

}

File: WEB2BEKEND/WEB2BEKEND/Models/Notification.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace WEB2BEKEND.Models
{
  public class Notification
  {
    [Key]
    public long Id { get; set; }


    public bool Deleted { get; set; }

    public bool Visible { get; set; }
    public string Type { get; set; }
    public string Text { get; set; }
    public string Status { get; set; }
    public string TimeStamp { get; set; }

    public User User { get; set; }
  }
}
